app/components/avo/views/resource_index_component.rb

```diff
@@ -29,6 +30,28 @@ def title
     end
   end
 
+  def view_type
+    @index_params[:view_type]
+  end
```

Comment:
This method was refactored and is not necessary, `view_type` can now be accessed from the resource: `resource.view_type`

```suggestion
```

Reply:
I pushed a commit that removes this method https://github.com/avo-hq/avo/pull/3695/commits/b0ea7501520a42e1deb0ed3486619f676773b01a